tftrt/examples/image-classification/image_classification.py

```diff
@@ -562,6 +615,8 @@ def get_frozen_graph(
         help='workspace size in bytes')
     parser.add_argument('--cache', action='store_true',
         help='If set, graphs will be saved to disk after conversion. If a converted graph is present on disk, it will be loaded instead of building the graph again.')
+    parser.add_argument('--mode', type=str, default='validation',
```

Comment:
Please add options for this so that argparse automatically checks if the users gives a wrong value.

Reply:
What should be the valid options?

Reply:
I would define two options for now: `validation` and `benchmark`.

Reply:
This is not done yet.